src/Notification/reducer.js

```diff
@@ -40,6 +40,31 @@ const notificationReducer = (state, action) => {
         notifications: state.notifications.filter(n => n.key !== action.key),
       };
 
+    case UPDATE_NOTIFICATION:
+      return {
+        ...state,
+        notifications: state.notifications.map(n => {
+          if (n.key === action.options.key) {
+            return {
+              ...n,
+              component: action.component,
+              options: {
+                autoDismiss: action.options.autoDismiss
+                  ? action.options.autoDismiss
+                  : state.options.autoDismiss,
```

Comment:
you can replace the ternary by `action.options.autoDismiss || state.options.autoDismiss` same for the others

Reply:
Done